logger/logger_test.go

```diff
@@ -121,6 +124,8 @@ func TestLogger_LogWithOptions(t *testing.T) {
 		assert.Equal(t, "anotherapp", line["app"])
 		assert.Equal(t, "production", line["env"])
 		assert.Equal(t, "error", line["level"])
+		assert.InDelta(t, time.Now().UnixNano()/int64(time.Millisecond), int64(line["timestamp"].(float64)), 5)
```

Comment:
I'm not sure why when I access line["timestamp"] it is of type float64? Am having to cast it in order to do this check here. Also have added the inDelta checks to all tests that make assertions on the values of other line fields